.travis.yml

```diff
@@ -50,6 +50,14 @@ before_script:
   # Remember the Drupal installation path.
   - DRUPAL_DIR=$(pwd)/drupal
 
+  # Reference OG in the Drupal site.
+  - cp -R $MODULE_DIR $DRUPAL_DIR/modules/og
```

Comment:
This is already taken care of, a symlink is created that puts the module in the drupal installation.